src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -1,164 +1,240 @@
-          </td>
-          <td>
-            {{ pool.stats.rd_bytes.rate | dimless }}
-          </td>
-          <td>
-            {{ pool.stats.rd.rate | dimless }} ops
-          </td>
-          <td>
-            {{ pool.stats.wr_bytes.rate | dimless }}
-          </td>
-          <td>
-            {{ pool.stats.wr.rate | dimless }} ops
-          </td>
-        </tr>
-        </tbody>
-      </table>
-    </cd-info-card>
-  </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <!-- LOGS -->
-      <div class="well"
-           *ngIf="contentData.clog || contentData.audit_log">
-        <fieldset>
-          <legend i18n>Logs</legend>
-
-          <tabset>
-            <tab heading="Cluster log"
-                 class="text-monospace"
-                 *ngIf="contentData.clog"
-                 i18n-heading>
-              <span *ngFor="let line of contentData.clog">
-                {{ line.stamp }}&nbsp;{{ line.priority }}&nbsp;
-                <span [ngStyle]="line | logColor">
-                  {{ line.message }}
-                  <br>
-                </span>
-              </span>
-            </tab>
-            <tab heading="Audit log"
-                 class="text-monospace"
-                 *ngIf="contentData.audit_log"
-                 i18n-heading>
-              <span *ngFor="let line of contentData.audit_log">
-                {{ line.stamp }}&nbsp;{{ line.priority }}&nbsp;
-                <span [ngStyle]="line | logColor">
-                  <span style="font-weight: bold;">
-                    {{ line.message }}
-                  </span>
-                  <br>
-                </span>
-              </span>
-            </tab>
-          </tabset>
-        </fieldset>
+      {{ contentData.pools.length }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Raw Capacity"
+                  class="cd-col-5"
+                  cardClass="card-medium"
+                  contentClass="card-chart"
+                  *ngIf="contentData.mon_status">
+      <cd-health-pie [data]="contentData"
+                     [isBytesData]="true"
+                     [displayLegend]="true"
+                     (prepareFn)="prepareRawUsage($event[0], $event[1])">
+      </cd-health-pie>
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Objects"
+                  class="cd-col-5"
+                  cardClass="card-medium"
+                  contentClass="content-medium content-highlight"
+                  *ngIf="contentData.df?.stats?.total_objects">
+      {{ contentData.df?.stats?.total_objects }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="PGs per OSD"
+                  class="cd-col-5"
+                  cardClass="card-medium"
+                  contentClass="content-medium content-highlight"
+                  *ngIf="contentData.pg_info">
+      {{ contentData.pg_info.pgs_per_osd | round }}
```

Comment:
How about using the `dimless` pipe instead, This would shorten the number for large clusters a lot.

Reply:
I agree.